config-linux.md

```diff
@@ -228,6 +186,46 @@ You can configure a container's cgroups via the `resources` field of the Linux c
 Do not specify `resources` unless limits have to be updated.
 For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
 
+#### Device whitelist
```

Comment:
Whitelist typically carries a positive connotation. In this case, this whitelist can be used to deny access to devices as well.